extensions/wikia/AdEngine3/src/ad-context.js

```diff
@@ -464,6 +464,12 @@ export default {
 			parameters: {},
 			timeout: 2000,
 		},
+		confiant: {
```

Comment:
Should it be exactly the same as on mobile?

Reply:
I asked Nef in ticket about this and we're planning to move QA of this task to Confiant, so they will verify that and we will change those values if required :)